include/deal.II/multigrid/multigrid.h

```diff
@@ -74,6 +74,60 @@ class Multigrid : public Subscriptor
     f_cycle
   };
 
+  /**
+   * A structure that has boost::signal objects for a number of actions that
+   * happen for the typical usage of this class.
+   *
+   * For documentation on signals, see
+   * http://www.boost.org/doc/libs/release/libs/signals2 .
+   */
+  struct Signals
+  {
+    /**
+     * This signal is triggered before and after the call to
+     * MGTransfer::copy_to_mg which transfers given vector to all levels.
+     */
+    boost::signals2::signal<void (bool start)> transfer_to_mg;
```

Comment:
here and below, make the argument to the signal `const`.

Reply:
I think here and for the other functions, you will also want to say something like "The argument `start` to the signal indicates whether the signal is currently being triggered <i>before</i> copying data to the individual multigrid levels (if `start` is `true`), or <i>after</i> copying data has been finished (if `start` is `false`)."

I think a reasonably educated reader could have inferred that, but my first reaction was "Oh, that's a bad idea to have the same signal executed before *and* after -- how are the called functions going to figure out which phase they're in?"